components/types.ts

```diff
@@ -0,0 +1,94 @@
+import { FieldError, UseFormRegister } from "react-hook-form";
+
+export type FormData = {
+  // id: number;
+  game_name: string;
+  type_of_game: string;
+  owned: string;
+  status: string;
+  platform: string;
+};
```

Comment:
is that actually needed? Seems that it's a shallow, less precise copy of `GameFormValues`